.devcontainer/bindings-ruby/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+	"name": "Rust with Ruby",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/ruby:1": {
+			"version": "latest"
+		}
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"Shopify.ruby-lsp"
+			]
+		}
+	},
+	"onCreateCommand": "pwd; cd bindings/ruby; bundle install"
+}
```

Comment:
`.devcontainer/bindings-ruby` files are for ruby users (like me) to easily join contributing.
You can add `.devcontainer/bindings-python` etc. This is official way on GitHub & vscode.